src/hotspot/share/utilities/ostream.cpp

```diff
@@ -404,7 +411,7 @@ char* stringStream::as_string(bool c_heap) const {
 }
 
 stringStream::~stringStream() {
-  if (_is_fixed == false && _buffer != NULL && _buffer != _small_buffer) {
+  if (_is_fixed == false && _buffer != _small_buffer) {
```

Comment:
Still using `_is_fixed == false` rather than `!_is_fixed`.

Reply:
Oh, I missed that, sorry. Will fix before integration.